### PR TITLE
Create FlightRouteAwards.php

### PR DESCRIPTION
**Pull request by Da-Geek**
This allows the Admins to create an Award that will be granted if a Pilot completes a flight with a defined
Departure and Arrival Airport.
So allowing an Award for a particular Flight.
This should solve issue #860

### CODE REVIEW

File: tests/AwardsTest.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ public function setUp(): void
     public function testGetAwardsClasses()
     {
         $classes = $this->awardSvc->findAllAwardClasses();
-        $this->assertCount(2, $classes);
+        $this->assertCount(3, $classes);
```
Comment (nabeelio) [inline tests/AwardsTest.php:35]: https://phpunit.readthedocs.io/en/9.3/assertions.html#assertgreaterthan

File: modules/Awards/Awards/FlightRouteAwards.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Modules\Awards\Awards;
+
+use App\Contracts\Award;
+
+/**
+ * Simple example of an awards class, where you can apply an award when a user
```
Comment (nabeelio) [inline modules/Awards/Awards/FlightRouteAwards.php:8]: I'd also update these docs

File: tests/AwardsTest.php (hunk 1 above)
Comment (Da-Geek) [reply tests/AwardsTest.php]: I found the documentation late last night and was looking to try some of these other methods

File: modules/Awards/Awards/FlightRouteAwards.php (hunk 2 above)
Comment (Da-Geek) [reply modules/Awards/Awards/FlightRouteAwards.php]: Just pushed bab3c55 to address this.

File: tests/AwardsTest.php (hunk 1 above)
Comment (Da-Geek) [reply tests/AwardsTest.php]: I went with assertGreaterThanOrEqual(2, $classes) in commit b804a0a, to include the 2 included Classes + any others that are added

**Pull request merged**